warehouse/oidc/models/github.py

```diff
@@ -132,7 +132,7 @@ class GitHubPublisherMixin:
         "job_workflow_ref": _check_job_workflow_ref,
     }
 
-    __required_unverifiable_claims__: set[str] = {"ref", "sha"}
+    __required_unverifiable_claims__: set[str] = {"ref", "sha", "jti"}
```

Comment:
Per note above: I think this should be in `OIDCPublisherMixin.__preverified_claims__` instead 🙂

Reply:
Updated in [5a65892](https://github.com/pypi/warehouse/pull/16254/commits/5a658928ff16e73a8e7ff5df2fe74422164e772d)